packages/reassure-measure/src/measure-renders.tsx

```diff
@@ -16,16 +16,31 @@ export interface MeasureOptions {
   warmupRuns?: number;
   wrapper?: React.ComponentType<{ children: React.ReactElement }>;
   scenario?: (screen: any) => Promise<any>;
+  writeFile?: boolean;
 }
 
-export async function measurePerformance(ui: React.ReactElement, options?: MeasureOptions): Promise<MeasureResults> {
-  const stats = await measureRender(ui, options);
-  await writeTestStats(stats, 'render');
+export async function measureRenders(ui: React.ReactElement, options?: MeasureOptions): Promise<MeasureResults> {
+  const stats = await measureRendersInternal(ui, options);
+
+  if (options?.writeFile !== false) {
+    await writeTestStats(stats, 'render');
+  }
 
   return stats;
 }
 
-export async function measureRender(ui: React.ReactElement, options?: MeasureOptions): Promise<MeasureResults> {
+/**
+ * @deprecated The `measurePerformance` function has been renamed to `measureRenders`. The `measurePerformance` alias is now deprecated and will be removed in future releases.
+ */
+export async function measurePerformance(ui: React.ReactElement, options?: MeasureOptions): Promise<MeasureResults> {
+  console.warn(
+    'The `measurePerformance` function has been renamed to `measureRenders`.\n\nThe `measurePerformance` alias is now deprecated and will be removed in future releases.'
+  );
```

Comment:
Can we call it once per test run? This will produce a lot of warnings that are fixable with one bulk search/replace